less/common.less

```diff
@@ -71,8 +71,8 @@ a.bold-link {
 a.more {
   font-weight: 600;
   &:after {
-    content: "\f30f";
-    font-family: "Ionicons";
+    content: "\f061";
+    font-family: "FontAwesome";
     margin: 0 .5rem;
   }
```

Comment:
These are the -> arrows seen on the Homepage, 

e.g.,
![screen shot 2015-06-16 at 2 58 51 pm](https://cloud.githubusercontent.com/assets/2896608/8195557/4603d864-1438-11e5-86a7-4ab95b2bbbd7.png)